File: app/src/main/java/com/example/getfood/ui/orderdetail/OrderDetailMvpPresenter.java
package com.example.getfood.ui.orderdetail;

import com.example.canteen_app_models.models.FullOrder;
import com.example.getfood.ui.base.BaseMvpPresenter;

public interface OrderDetailMvpPresenter<V extends OrderDetailMvpView> extends BaseMvpPresenter<V> {
    void fetchOrderDetails(FullOrder fullOrder);

    void setRatingValueForOrderItem(float rating, int position, String order);
}

File: app/src/main/java/com/example/getfood/ui/orderdetail/OrderDetailMvpPresenter.kt
package com.example.getfood.ui.orderdetail

import com.example.canteen_app_models.models.FullOrder
import com.example.getfood.ui.base.BaseMvpPresenter

interface OrderDetailMvpPresenter<V : OrderDetailMvpView?> : BaseMvpPresenter<V> {
    fun fetchOrderDetails(fullOrder: FullOrder)
    fun setRatingValueForOrderItem(rating: Float, position: Int, order: String?)
}